mungegithub/mungers/flake-manager.go

```diff
@@ -127,6 +127,11 @@ func (p *FlakeManager) isIndividualFlake(f cache.Flake) bool {
 	return true
 }
 
+// makeGubernatorLink returns a URL to view the build results in a GCS path.
+func makeGubernatorLink(id string) string {
+	return "https://k8s-gubernator.appspot.com/build" + id
+}
```

Comment:
Oops. That's weird, I tested it, is that job an exception?

Why not just fix the link?

On Wed, Jun 1, 2016 at 3:12 PM, Ryan Hitchman <notifications@github.com>
wrote:

> In mungegithub/mungers/flake-manager.go
> <https://github.com/kubernetes/contrib/pull/1112#discussion_r65452773>:
>
> > @@ -127,6 +127,11 @@ func (p *FlakeManager) isIndividualFlake(f cache.Flake) bool {
> >  	return true
> >  }
> >
> > +// makeGubernatorLink returns a URL to view the build results in a GCS path.
> > +func makeGubernatorLink(id string) string {
> > +	return "https://k8s-gubernator.appspot.com/build" + id
> > +}
>
> The direct link is incorrect, linking to things like
> https://storage.googleapis.com/kubernetes-jenkins/logs/kubernetes-e2e-gce/17823/
> which is a 404.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1112/files/045e17ab2b998712c9d04de10fb3180206085e99#r65452773>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAngls6K5o1cXFVmEfbd2EFtdQSPUD1Kks5qHgPTgaJpZM4IrErK>
> .
>